src/lib/api/group/client/groupApiClient.client.ts

```diff
@@ -2,38 +2,78 @@
 
 import { request } from '@/lib/api/common/client';
 import { IGroup } from '@/types/definition';
+
+import { Err, Ok, Result } from 'result-ts-type';
 const BACKEND_API_DOMAIN = process.env.NEXT_PUBLIC_BACKEND_API_DOMAIN || '';
 
 /**
- * Interface representing the Params for the method to create a group.
+ * Interface representing the Payload for the method to create a group.
  * @property groupName - The name of the group.
  */
-export interface ICreateGroupParams {
+export interface IPostCreateGroupPayload {
   groupName: IGroup['name'];
 }
 
 /**
  * Interface representing the API response for the method to create a group.
  * @property groupId - The unique identifier of the group.
  */
-export interface ICreateGroupApiResponse {
+export interface IPostCreateGroupApiResponse {
   groupId: IGroup['id'];
 }
 
 /**
- * Create a new group.
- * @param params - The parameters for the group to be created.
- * @returns The ID of the newly created group.
+ * Method to create a group.
+ * @param payload - The payload for the method.
+ * @returns The unique identifier of the group.
  */
-export const createGroup = async (params: ICreateGroupParams): Promise<string> => {
+export const postCreateGroup = async (
+  payload: IPostCreateGroupPayload,
```

Comment:
Just changed the variable name from `params` to `payload` because it's defined as `payload` in web api doc